jest.config.js

```diff
@@ -11,10 +11,10 @@ module.exports = {
   collectCoverage: true,
   coverageThreshold: {
     global: {
-      branches: 80,
-      functions: 80,
-      lines: 80,
-      statements: 80,
+      branches: 88,
+      functions: 100,
+      lines: 95,
+      statements: 95,
```

Comment:
After these changes, is it safe to remove the specific coverage for `APIBasedAuth.ts`?